resharper/resharper-unity/src/Rider/UnityYamlOccurenceProvider.cs

```diff
@@ -25,6 +26,7 @@ public IOccurrence MakeOccurrence(FindResult findResult)
                 
                 if (reference != null)
                 {
+                    var cache = reference.ComponentDocument.GetSolution().GetComponent<UnitySceneDataCache>();
```

Comment:
todo for 19.3. [OccurenceProviders] should be composed via SolutionFeaturePartsContainer, so such components could just inject solution and do not do .GetComponent on every occurrence create.